dev/.rat-excludes

```diff
@@ -102,7 +102,7 @@ spark-deps-.*
 org.apache.spark.scheduler.ExternalClusterManager
 .*\.sql
 .Rbuildignore
-org.apache.spark.deploy.yarn.security.ServiceCredentialProvider
+org.apache.spark.deploy.security.ServiceCredentialProvider
```

Comment:
In a change I have internally I changed this to:

```
META-INF/services/*
```

If that works (haven't tested yet) I think that's better than having to keep changing / adding these files here...